### DISCUSSION TITLE
Premium A+ Content API Support

### DISCUSSION THREAD

**Discussion by bbenda02** [amzn/selling-partner-api-models#582] [Ideas]
Can we please allow for the creation of premium a+ content in the API? Regular a+ content blocks do not display images in the best quality, and it's counterpart premium a+ does. It would really allow customers to see better images on listings and improve their shopping experience. Thanks.

**Comment by MeenaAmz** [comment]
Thanks for the suggestion. Yes, this has been already requested as an enhancement. I will add your feedback to the request as well.
